stable/weave-scope/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Where defaults exist, the values are set to them here.
+# Values with no preferred or common defaults are set to empty strings.
+global:
+# global.image: (required) the image that will be used for this release
```

Comment:
Please indent comments as well to make things clearer.

Reply:
What's the preferred format?

```
# foo: comment
foo: 1
bar:
  # bar.baz: comment
  baz: 2
```

or something else?